painttiming.bs

```diff
@@ -177,8 +187,8 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute's getter must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute's getter must return 0.
 
-NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
-This allows developers to detect support for paint timing.
+NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} where the [=relevant global object=] is associated with a [=paint-timing eligible=] [=browsing context=].
```

Comment:
How is this one @yoavweiss ?

Reply:
I don't think there's a direct association between a global object and a browsing context.
Also, a relevant global object is relevant to a certain object.

Would the following work?
"A user agent implementing PerformancePaintTiming would need to include "paint" in a [frozen array of supported entry types](https://www.w3.org/TR/performance-timeline-2/#dfn-frozen-array-of-supported-entry-types) |array| when |array|'s [relevant settings object](https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object) has a [responsible Document](https://html.spec.whatwg.org/multipage/webappapis.html#responsible-document) |doc|, and |doc|'s [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#concept-document-bc) is non-null and [=paint-timing eligible=]"

Reply:
I think I got a more succinct version that would also work:
"A user agent implementing PerformancePaintTiming would need to include "paint" in a [frozen array of supported entry types](https://www.w3.org/TR/performance-timeline-2/#dfn-frozen-array-of-supported-entry-types) |array| when |array|'s [relevant settings object](https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object)'s [responsible browsing context](https://html.spec.whatwg.org/multipage/webappapis.html#responsible-browsing-context) is [=paint-timing eligible=]"